content/notes/concepts.md

```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
 
 
 * **Access**: optimizing for social welfare will mean selecting a business model which prioritizes high levels of access. By contrast, business models which use closed rights frameworks may enable you to make more money (although this is not always the case, as businesses such as Facebook demonstrate). 
-* **Quality:** optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. This is a matter of cost-benefit analysis, trading off gains in quality most often against levels of provision (i.e. the same resources can get you more lower quality digital goods or fewer higher quality digital goods) By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
+* **Quality**: optimizing for social welfare will mean setting the level of data quality at the most useful level affordable, all things being equal. By contrast, a business model seeking to make the most money possible will produce the digital goods at whatever quality maximizes profit. Often, this will be the lowest quality possible without detrimentally impacting demand.
 * **Profitability**: different models will be differently profitable. Crucially, profitability can often seem to come into conflict with access. Optimizing for money making via profit may mean the use of fees and other mechanisms which reduce access and thus social welfare. On the flip side, the most obvious route to high access and thus social welfare–free universal public provision–is innately unprofitable by definition. 
```

Comment:
Not just unprofitable but may lack revenue ...

Strictly, from a social welfare point of view profitability isn't really a feature at all ... you'd only have access and quality ... (i wonder therefore if we drop profitability for now ...?)